x-pack/filebeat/docs/inputs/input-netflow.asciidoc

```diff
@@ -142,6 +142,34 @@ value or one of the named ranges supported by the
 <<condition-network, `network`>> condition. The default value is `[private]`
 which classifies RFC 1918 (IPv4) and RFC 4193 (IPv6) addresses as internal.
 
+[float]
+=== Metrics
+
+This input exposes metrics under the <<http-endpoint, HTTP monitoring endpoint>>.
+These metrics are exposed under the `/inputs/` path. They can be used to
+observe the activity of the input.
+
+You must assign a unique `id` to the input to expose metrics.
+
+[options="header"]
+|=======
+| Metric                         | Description
+| `device`                       | Host/port of the UDP stream.
+| `udp_read_buffer_length_gauge` | Size of the UDP socket buffer length in bytes (gauge).
+| `received_events_total`        | Total number of packets (events) that have been received.
+| `received_bytes_total`         | Total number of bytes received.
+| `receive_queue_length`         | Aggregated size of the system receive queues (IPv4 and IPv6) (linux only) (gauge).
+| `system_packet_drops`          | Aggregated number of system packet drops (IPv4 and IPv6) (linux only) (gauge).
+| `arrival_period`               | Histogram of the time between successive packets in nanoseconds.
+| `processing_time`              | Histogram of the time taken to process packets in nanoseconds.
+| `discarded_events_total`       | Total number of discarded events.
+| `decode_errors_total`          | Total number of errors at decoding a packet.
+| `flows_total`                  | Total number of received flows.
+| `open_connections`             | Number of current active netflow sessions.
+|=======
+
+Histogram metrics are aggregated over the previous 1024 events.
+
 [id="{beatname_lc}-input-{type}-common-options"]
```

Comment:
IMO the "Metrics" should be last on the page. Or at least not in the middle of the input specific config options and the common config options.